### PR TITLE
Change the tag for performance tests.

### PR DESCRIPTION
**Pull request by gmarek**
cc @wojtek-t 

Partially addresses #19542

### CODE REVIEW

File: test/e2e/density.go [hunk 1]
```diff
@@ -190,8 +190,10 @@ var _ = Describe("Density [Skipped]", func() {
 	for _, testArg := range densityTests {
 		name := fmt.Sprintf("should allow starting %d pods per node", testArg.podsPerNode)
 		if testArg.podsPerNode == 30 {
-			name = "[Performance] " + name
+			name = "[Feature:Performance] " + name
 			framework.addonResourceConstraints = density30AddonResourceVerifier()
+		} else {
+			name = "[Skipped] " + name
```
Comment (ihmccreery) [inline test/e2e/density.go]: The effort here is to remove the `[Skipped]` label from existence completely.  Why do these need to be skipped?

**Comment by ihmccreery** [comment]
Thanks @gmarek; please see comment.

File: test/e2e/density.go (hunk 1 above)
Comment (gmarek) [inline test/e2e/density.go]: We have some predefined types of density tests: 3, 30 and 100. 3 and 100 are for manual runs only and we don't want to run them on Jenkins, as they either don't provide any value (3), or are above our current SLO for density (100). It's still useful to have them defined to make running them easy (I use density 3 pretty often to check various stuff, and density 100 is/was run by RedHat and/or Samsung @smarterclayton @spiffxp). I add a flag to define density, but it'll probably make test scripts even worse. 

We can change Skipped to Manual to better reflect the meaning. WDYT?

Comment (smarterclayton) [inline test/e2e/density.go]: @timothysc on density

Comment (timothysc) [inline test/e2e/density.go]: @gmarek I would like to default enable 100 for 1.2 vs. Manual|Skipped.

Comment (ihmccreery) [inline test/e2e/density.go]: > We can change Skipped to Manual to better reflect the meaning. WDYT?

I'd rather avoid manual running of tests as something that's expected.

> as they either don't provide any value (3)

Why do we still have the (3) test at all then?

> or are above our current SLO for density (100)

How about `[Feature:ExperimentalPerformance]` or something like that?  That sets the expectation that you all might eventually run this in an 'experimental' job that isn't expected to be stable, or something.

Comment (gmarek) [inline test/e2e/density.go]: Well, everything's stable. Usually. The tests are with lower densities are just faster to run, which is really, really, really, really important for development. I treat it as smoke tests and I'd rather keep them. I'm OK with having different label for 100 pod density, and just not running it in any suite (but I guess it's pretty much equivalent to having a label Skipped/Manual).

Comment (ihmccreery) [inline test/e2e/density.go]: > I'm OK with having different label for 100 pod density, and just not running it in any suite (but I guess it's pretty much equivalent to having a label Skipped/Manual).

Let's do that.  The difference is that we can scan through the `[Feature:...]`s and make sure they're running where we expect.

I'll let you & @timothysc decide about 100 enabled, etc.

Comment (timothysc) [inline test/e2e/density.go]: I'm ok with Manual for now, unto our official SLO's say 100. 

/cc @dchen1107

**Comment by gmarek** [comment]
@ihmccreery PTAL

**Comment by ihmccreery** [comment]
LGTM, thanks Marek!

**Pull request merged**